livre/core/Render/GLContext.h

```diff
@@ -43,6 +43,8 @@ class GLContext : public boost::enable_shared_from_this< GLContext >
      */
     void shareContext( GLContextPtr dstContextPtr );
 
+    const GLEWContext* glewGetContext() const;
```

Comment:
It's the GL way of a include statement in my opinion. You cannot use something that is not defined/included, like using a std::vector w/o including it. It's not super nice, agreed, but the way GL is currently used in Livre this right now is a way to have correct code.